Automattic/jetpack
# Social: Remove instagram welcome notice

Fixes #35433

We do not need this notice anymore as Instagram is in for quite some time now, so we get rid of it with this change, which will also solve the linked issue


### Other information:


## Jetpack product discussion


#35433
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* On trunk go to your local site, and delete your Instagram connection if you have any
* Run `jetpack docker wp option delete jetpack_social_dismissed_notices` to unset the dismissed notices
* Create a new post, and in the Jetpack sidebar you should see the Instagram notice, which you cannot close.
![CleanShot 2024-02-22 at 13 43 29 png](https://github.com/Automattic/jetpack/assets/36671565/fdd6abc9-f63c-4356-a3a5-323cd46e3a60)
* Apply this patch, and run `jetpack docker wp option delete jetpack_social_dismissed_notices` again if you tried to close it.
* Make sure that the notice won't appear.
